plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityExploredLocations.java

```diff
@@ -0,0 +1,45 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.Location;
+import org.bukkit.entity.Sniffer;
+
+public class EntityExploredLocations extends EntityProperty<ListTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name explored_locations
+    // @input ListTag(LocationTag)
+    // @description
+    // If the entity is a sniffer, controls the locations that it has explored. 

```

Comment:
Property meta format (needs to make sense for both the tag and mech) - probably something like `A sniffer's explored locations`? `The locations a sniffer explored`?